### PR TITLE
CPLP-1241 add DateLastChanged to app

### PR DESCRIPTION
**Pull request by Phil91**

### CODE REVIEW

File: postgres/init/21-portal-basedata.sql [hunk 1]
```diff
@@ -478,6 +478,14 @@ COPY portal.notification_type (id, label) FROM stdin;
 \.
 
 
+--
+-- Data for Name: notifications; Type: TABLE DATA; Schema: portal; Owner: portal
+--
+
+COPY portal.notifications (id, receiver_user_id, date_created, content, notification_type_id, is_read, due_date, creator_user_id) FROM stdin;
```
Comment (ntruchsess) [inline postgres/init/21-portal-basedata.sql:485]: notifications certainly belong to content, not basedata

File: postgres/init/20-portal.sql [hunk 2]
```diff
@@ -211,7 +211,8 @@ CREATE TABLE portal.apps (
     contact_number character varying(255),
     provider character varying(255) NOT NULL,
     provider_company_id uuid,
-    app_status_id integer NOT NULL
+    app_status_id integer NOT NULL,
+    date_last_changed timestamp with time zone
```
Comment (ntruchsess) [inline postgres/init/20-portal.sql:215]: the corresponding change in 22-portal-content.sql is missing.

**Review by ntruchsess — CHANGES_REQUESTED** [review]

File: postgres/init/20-portal.sql (hunk 2 above)
Comment (Phil91) [reply postgres/init/20-portal.sql:215]: added the missing column to the copy statement

File: postgres/init/21-portal-basedata.sql (hunk 1 above)
Comment (Phil91) [reply postgres/init/21-portal-basedata.sql]: moved it.

File: backup_portal_basedata.ps1 [hunk 3]
```diff
@@ -40,7 +40,7 @@ cd $PSScriptRoot
 
 # Set environment variables for the database
 $DEVENV_IP = '172.19.0.2'
-$BASETABLES='(agreement_categories|app_subscription_statuses|app_statuses|company_application_statuses|company_role_descriptions|company_roles|company_service_account_statuses|company_statuses|company_user_statuses|connector_statuses|connector_types|consent_statuses|countries|document_types|document_status|identity_provider_categories|invitation_statuses|languages|notifications|notification_type|use_cases)'
+$BASETABLES='(agreement_categories|app_subscription_statuses|app_statuses|company_application_statuses|company_role_descriptions|company_roles|company_service_account_statuses|company_statuses|company_user_statuses|connector_statuses|connector_types|consent_statuses|countries|document_types|document_status|identity_provider_categories|invitation_statuses|languages|notification_type|use_cases)'
```
Comment (ntruchsess) [inline backup_portal_basedata.ps1:43]: we certainly need to include table notifications in 22-portal-content.sql. To do so please update 'backup_portal_data.sh' and create corresponding powershell-script if you cannot run the sh-script on wsl

**Review by ntruchsess — CHANGES_REQUESTED** [review]

**Review by ntruchsess — APPROVED** [review]

**Pull request merged**